rc/targets.go

```diff
@@ -149,6 +150,7 @@ func CommandTargetClient(selectedTarget TargetName, commandInsecure *bool) (conc
 		Dial: (&net.Dialer{
 			Timeout: 10 * time.Second,
 		}).Dial,
+		Proxy: http.ProxyFromEnvironment,
```

Comment:
I set the proxy according to the default transport client, see: https://golang.org/src/net/http/transport.go#L34 (line 34)